features/users/settings/notification-types/configureNotificationTypes.ts

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+
+type ConfiguredNotificationTypes = {
+  codes: string[];
+};
+
+function configureNotificationTypesForUser(userId: string) {
+  return (notifications: ConfiguredNotificationTypes) => {
```

Comment:
You can name this configurationTypes to be consistent with method name.

Reply:
Done.